test/SlowTests/Sharding/Client/Operations/GetDocumentConflictsTests.cs

```diff
@@ -60,13 +60,13 @@ public async Task CanGetReplicationDocumentConflicts()
                     Assert.Equal(expected, totalResults);
 
                     var documentsConflict = await store.Maintenance.SendAsync(new GetConflictsOperation(start: 0));
-                    Assert.Equal(20, documentsConflict.Results.Length);
+                    Assert.Equal(expected, documentsConflict.Results.Count);
 
-                    documentsConflict = await store.Maintenance.SendAsync(new GetConflictsOperation(start: 12));
-                    Assert.Equal(8, documentsConflict.Results.Length);
+                    documentsConflict = await store.Maintenance.SendAsync(new GetConflictsOperation(start: 2));
```

Comment:
`pageSize` change I understand, but why the `start` reduced to `2` from `12`?

Reply:
because now we have 10 unique results instead of 20